qemu/tests/win_nics_teaming.py

```diff
@@ -0,0 +1,114 @@
+import logging
+import time
+import os
+import random
+
+from avocado.utils import process
+from virttest import utils_net
+from virttest import utils_misc
+
+
+def run(test, params, env):
+    """
+    Nic teaming test in guest.
+
+    1) Start guest with four nic devices.
+    2) Setup Team in guest.
+    3) Execute file transfer from host to guest.
+    4) Repeatedly set enable/disable interfaces by 'netsh interface set'
+    5) Execute file transfer from guest to host.
+
+    :param test: Kvm test object.
+    :param params: Dictionary with the test parameters.
+    :param env: Dictionary with test environment.
+    """
+    def copy_file_from_host(host_path, guest_path, vm):
+        # transfer data
+        logging.info("Transfer data from host to %s" % vm.name)
+        vm.copy_files_to(host_path, guest_path)
+
+    def copy_file_from_guest(guest_path, host_path, vm):
```

Comment:
Only a single method is invoked, I don't think we need to define these two functions.

Reply:
I have removed these two functions.